src/aws-cur/cost-report.ts

```diff
@@ -0,0 +1,102 @@
+import { Stack, aws_cur, aws_iam, aws_s3 } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+
+export class ReportGranularity {
+
+  public static readonly HOURLY: ReportGranularity = new ReportGranularity('HOURLY');
+  public static readonly DAILY: ReportGranularity = new ReportGranularity('DAILY');
+  public static readonly MONTHLY: ReportGranularity = new ReportGranularity('MONTHLY');
+
+  private constructor(public readonly value: string) { }
+}
+
+export class CurFormat {
+
+  public static readonly TEXT: CurFormat = new CurFormat('GZIP', 'textORCsv');
```

Comment:
Isn't `textORcsv` instead of `textORCsv`?

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cur-reportdefinition.html#cfn-cur-reportdefinition-format

https://docs.aws.amazon.com/ja_jp/aws-cost-management/latest/APIReference/API_cur_ReportDefinition.html#awscostmanagement-Type-cur_ReportDefinition-Format

```suggestion
  public static readonly TEXT: CurFormat = new CurFormat('GZIP', 'textORcsv');
```

Reply:
Very good catch.

Reply:
Also, I thought the variable name `TEXT_OR_CSV` (not `TEXT`) would be easier to understand as it follows the information output to the management console, what do you think?